content/docs/rfcs/14/README.md

```diff
@@ -1,120 +1,102 @@
+## Message Attributes
 
-The `contentTopic` field SHOULD be filled out to allow for content-based filtering.
-See [12/WAKU2-FILTER](/spec/12) and [13/WAKU2-STORE](/spec/13) for more details.
-To enable a bidirectional bridge with Waku v1 see [15/WAKU2-BRIDGE](/spec/15) for further requirements on this field.
 
-The `version` field MAY be filled out to allow for various types of payload encryption.
-Omitting it means the version is 0.
+The `payload` attribute SHOULD contain the data to be sent.
 
-The `timestamp` field MAY be filled out to signify the time at which the message is generated by its sender. 
-This field holds the Unix epoch time in nanoseconds. 
-Omitting it means the timestamp is unspecified.
+The `contentTopic` attribute SHOULD specify a string identifier that can be used for content-based filtering.
 
-The `ephemeral` field MAY be set to signify the transient nature of the message.
-If the message should be stored by the [store protocol](/spec/13), then this field MUST be set to `false`, which is equivalent to omitting the field.
-If the message should not be stored by the [store protocol](/spec/13), then this field MUST be set to `true`.
+The `version` attribute MAY contain a version number to discriminate different types of payload encryption. If omitted, the value SHOULD be interpreted as version 0.
 
-See [13/WAKU2-STORE](/spec/13) for more details.
+The `timestamp` attribute signifies the time at which the message was generated by its sender. This field SHOULD contain the Unix epoch time in nanoseconds. If the attribute is omitted, it SHOULD be interpreted as timestamp 0.
 
-## Payloads
+The `ephemeral` attribute signifies the transient nature of the message. If this field is set to `true`, the message SHOULD be interpreted as ephemeral. Else, if the field is omitted or set to `false`, the message SHOULD be interpreted as non-ephemeral.
 
-Payloads are implemented using [protocol buffers v3](https://developers.google.com/protocol-buffers/).
+# Wire Format
+
+The `WakuMessage` wire format is specified using [protocol buffers v3](https://developers.google.com/protocol-buffers/).
 
 ```protobuf
 syntax = "proto3";
 
 message WakuMessage {
   bytes payload = 1;
-  string contentTopic = 2;
+  string content_topic = 2;
   uint32 version = 3;
   sint64 timestamp = 10;
   bool ephemeral = 31;
 }
 ```
 
-## Payload encryption
-
-Payload encryption depends on the `version` field.
-
-### Version 0
+# Payload encryption
 
-This indicates that the payload SHOULD be either unencrypted or that encryption is done at a separate layer outside of Waku.
+The waku message payload MAY be encrypted. The message `version` attribute indicates the schema used to encrypt the payload data.
 
-### Version 1
+- **Version 0:**  The payload SHOULD be interpreted as unencrypted; additionally, it CAN indicate that the message payload has been encrypted at the application layer.
```

Comment:
```suggestion
- **Version 0:**  The payload SHOULD be interpreted as unencrypted; additionally, it MAY indicate that the message payload has been encrypted at the application layer.
```

`CAN` is not a [recognised RFC keyword](https://rfc.vac.dev/spec/1/#language).